target/linux/ath79/image/generic.mk

```diff
@@ -999,6 +999,20 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/meraki_mr16
+  SOC := ar7161
+  DEVICE_VENDOR := Meraki
+  DEVICE_MODEL := MR16
+  ROOTFS_SIZE := 13440k
+  IMAGE_SIZE := 15680k
+  DEVICE_PACKAGES := kmod-spi-gpio kmod-owl-loader
+  IMAGE/sysupgrade.bin := append-rootfs | pad-rootfs | \
+	pad-to $$$$(ROOTFS_SIZE) | append-kernel | append-metadata | check-size
+  IMAGES := sysupgrade.bin
```

Comment:
IMAGES is set to the default and can be dropped.

Reply:
Done.